unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/BodyShapeController.cs

```diff
@@ -221,7 +221,7 @@ private void InitializeAvatarAudioAndParticleHandlers(GameObject container, Anim
     {
         //NOTE(Mordi): Adds handler for animation events, and passes in the audioContainer for the avatar
         AvatarAnimationEventHandler animationEventHandler = createdAnimation.gameObject.AddComponent<AvatarAnimationEventHandler>();
-        AudioContainer audioContainer = container.transform.parent.parent.GetComponentInChildren<AudioContainer>();
+        AudioContainer audioContainer = container.transform.parent.parent.parent.GetComponentInChildren<AudioContainer>();
```

Comment:
parent.parent.parent doesn't look, can we try to get gameobject reference that has the AudioContainer somehow?

Reply:
It doesn't really matter, this is going to get removed once the new avatar merges (which is imminent)

Reply:
Yeah i've seen that ugly thing, I looked a bit around and it felt like there was the need of a big refactoring to avoid that parent chain, I didn't really feel confident in making big changes to avoid that, if the avatar merge solves it perfect, otherwise I'll open a technical debt issue with that